backend/users/views.py

```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from django.contrib.auth import login, authenticate
+from django.shortcuts import render, redirect
 
-# Create your views here.
+from users.forms import SignUpForm
+
+
+def signup(request):
+    if request.method == 'POST':
+        form = SignUpForm(request.POST)
+        if form.is_valid():
+            form.save()
+            username = form.cleaned_data.get('username')
+            raw_password = form.cleaned_data.get('password1')
+            user = authenticate(username=username, password=raw_password)
+            login(request, user)
```

Comment:
We don't want this kind of custom logic, Django already has ways to signup users, please don't duplicate it